internal/xds/v3/contour.go

```diff
@@ -81,7 +82,7 @@ func (s *contourServer) stream(st grpcStream) error {
 	done := func(log logrus.FieldLogger, err error) error {
 		// If the stream has been closed by the client "gracefully",
 		// do not log as an error.
-		if err != nil && err != context.Canceled && status.Code(err) != codes.Canceled {
+		if err != nil && !errors.Is(err, context.Canceled) && status.Code(err) != codes.Canceled {
```

Comment:
can we revert this change since this unrelated to the goal of this PR? I am worried that if there is something wrong here it would be hard to parse through the commit history

Reply:
done